### PR TITLE
Dedicated host support

### PR DESCRIPTION
**Pull request by jrroman**
These are mostly test values for debugging cloudinit issues for macos

vpc-id and subnet-id stated to pin kip to an AZ for dedicated hosts

### CODE REVIEW

File: deploy/manifests/kip/base/cloudinit.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+anka license activate -f $ANKA_LICENSE
+anka license accept-eula || true
```
Comment (hidalgopl) [inline deploy/manifests/kip/base/cloudinit.sh:4]: I believe that we need to run those anka commands with `sudo`

File: deploy/manifests/kip/base/cloudinit.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+anka license validate
+
+wget -O /usr/local/bin/itzo https://itzo-kip-dev-download.s3.amazonaws.com/itzo-darwin-818
```
Comment (hidalgopl) [inline deploy/manifests/kip/base/cloudinit.sh:7]: wget isn't installed, let's use curl

**Comment by jrroman** [comment]
Sorry @hidalgopl this is not actually a real PR it was just reflecting the changes necessary for potentially using a secondary mac cloudinit file. More so to reflect that vilmos could see changes on my local

File: deploy/manifests/kip/base/cloudinit.sh (hunk 1 above)
Comment (jrroman) [reply deploy/manifests/kip/base/cloudinit.sh:4]: if it were executed by mac os init it would be executed as a root user

**Pull request closed without merging**